language/move-compiler/src/parser/lexer.rs

```diff
@@ -638,3 +637,68 @@ fn get_name_token(name: &str) -> Tok {
         _ => Tok::Identifier,
     }
 }
+
+// Trim the whitespace characters, include: space, tab, lf(\n) and crlf(\r\n).
+fn trim_whitespace(text: &str) -> &str {
+    let mut pos = 0;
+    let mut iter = text.chars();
+
+    while let Some(chr) = iter.next() {
+        match chr {
+            ' ' | '\t' | '\n' => pos += 1,
+            '\r' => {
+                let nch = iter.next();
+                if nch.is_some() && nch.unwrap() == '\n' {
```

Comment:
Should rewrite to

```
if let Some('\n') = nch {
}
```
, is more simple?

Reply:
Good

Reply:
@jolestar updated!